addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -468,6 +482,19 @@ const actions = {
         env.call('bus_service', 'startPolling');
         dispatch('_startLoopFetchPartnerImStatus');
     },
+    /**
+     * Insert some text content in an other.
+     *
+     * @param {string} initialContent
+     * @param {string} addedContent
+     * @param {integer} selectionStart
+     * @param {integer} selectionEnd
+     */
+    insertTextContent({}, initialContent, addedContent, selectionStart, selectionEnd) {
```

Comment:
This does not manipulate store, can be moved to utils.

Reply:
Done